pkgs/development/libraries/pmdk/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, pkgconfig, autoconf, automake, doxygen, graphviz, pandoc
+}:
+
+stdenv.mkDerivation rec {
+       name="pmdk-1.4.1";
```

Comment:
If you declare the version separately like `version = "1.4.1"` you can use it for the revision in `fetchFromGitHub` like `rev = version` and of course in `name` like `name ="pmdk-${version}`".

Reply:
Thanks.